GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -4829,8 +4919,37 @@ private async Task<(bool useLongValueColumn, bool valueChanged, bool deleteValue
                                || !String.Equals(previousField.GroupName, wiserItemDetail.GroupName, StringComparison.OrdinalIgnoreCase)
                                || !String.Equals(previousField.LanguageCode, wiserItemDetail.LanguageCode, StringComparison.OrdinalIgnoreCase);
             }
+            
+            if (alsoSaveSeoValue)
+            {
+                DataTable queryResult;
 
-            return (useLongValueColumn, valueChanged, deleteValue, alsoSaveSeoValue);
+                if (wiserItemDetail.IsLinkProperty && wiserItemDetail.ItemLinkId > 0)
+                {
+                    queryResult = await databaseConnection.GetAsync($@"SELECT id 
+FROM {WiserTableNames.WiserItemLinkDetail} 
```

Comment:
WiserItemLinkDetail kan ook een prefix hebben in de GCL, graag dit nog toevoegen. Dit wordt al op plaatsen in de code gedaan waar dit afgekeken kan worden.

Reply:
https://app.asana.com/0/1200346761113317/1204562731570674/f